Sources/InfiniteScroll/InfiniteScrollView.swift

```diff
@@ -78,6 +98,12 @@ public extension InfiniteScrollView {
 }
 
 public extension InfiniteScrollView where GroupView == ForEach<[T], T.ID, InfiniteScrollGroupContent<T, ItemView>> {
+  /// This function creates an instance of InfiniteScrollView with the provided parameters.
+  /// - Parameters:
+  ///   - pageInfo: The information about the page. Default value is PageInfo.default.
+  ///   - loadPage: A closure that loads the page asynchronously. It throws an error if the loading fails.
```

Comment:
A closure that is called by InfiniteScrollView, when it requests data for the next page ...